integration-tests/suites/base.go

```diff
@@ -186,6 +186,12 @@ func (s *IntegrationTestSuiteBase) execContainer(containerName string, command [
 	return s.executor.Exec(cmd...)
 }
 
+func (s *IntegrationTestSuiteBase) execContainerShellScript(containerName string, script string, args ...string) (string, error) {
+	cmd := []string{"docker", "exec", "-i", containerName, "/bin/sh", "-s"}
```

Comment:
Do we want to enforce usage of /bin/sh here? Would be nice to have the option to use /bin/bash or others.

Reply:
You are right, however, I have no idea how to implement that option. Would you have suggestions ?

Reply:
You could try to make it an optional argument. You might not be able to do that, because of the variadic parameter. You might do something like

func (s *IntegrationTestSuiteBase) execContainerShellScriptWithShell(containerName string, script string, shell string, args ...string) (string, error) {
	cmd := []string{common.RuntimeCommand, "exec", "-i", containerName, shell, "-s"}
	cmd = append(cmd, args...)
	return s.executor.ExecWithStdin(script, cmd...)
}

func (s *IntegrationTestSuiteBase) execContainerShellScript(containerName string, script string, args ...string) (string, error) {
	return s.execContainerShellScriptWithShell(containerName, script, "/bin/sh", args...)
}

Reply:
This suggestion has been implemented. My concern is that all the parameter are strings, including the variadic part; this might be quite error prone if the caller misses one of the initial parameters, since the compiler will be just happy with it :-/